sos/report/plugins/kdump.py

```diff
@@ -42,6 +42,11 @@ class RedHatKDump(KDump, RedHatPlugin):
     files = ('/etc/kdump.conf',)
     packages = ('kexec-tools',)
 
+    option_list = [
+        PluginOpt("get_vm_core", default=False, val_type=bool,
```

Comment:
can we change this to use hyphens, as per the global changes that were made a few days ago

Reply:
Done